taskt/Core/Automation/Commands/Loop Commands/BeginLoopCommand.cs

```diff
@@ -1,52 +1,57 @@
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Date Compare")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Variable Compare")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Variable Has Value")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Variable Is Numeric")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Window Name Exists")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Active Window Name Is")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("File Exists")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Folder Exists")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Web Element Exists")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("GUI Element Exists")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Error Occured")]
-        [Attributes.PropertyAttributes.PropertyUISelectionOption("Error Did Not Occur")]
-        [Attributes.PropertyAttributes.InputSpecification("Select the necessary comparison type.")]
-        [Attributes.PropertyAttributes.SampleUsage("Select **Value**, **Window Name Exists**, **Active Window Name Is**, **File Exists**, **Folder Exists**, **Web Element Exists**, **Error Occured**")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Loop Condition")]
+        [PropertyUISelectionOption("Value Compare")]
+        [PropertyUISelectionOption("Date Compare")]
+        [PropertyUISelectionOption("Variable Compare")]
+        [PropertyUISelectionOption("Variable Has Value")]
+        [PropertyUISelectionOption("Variable Is Numeric")]
+        [PropertyUISelectionOption("Window Name Exists")]
+        [PropertyUISelectionOption("Active Window Name Is")]
+        [PropertyUISelectionOption("File Exists")]
+        [PropertyUISelectionOption("Folder Exists")]
+        [PropertyUISelectionOption("Web Element Exists")]
+        [PropertyUISelectionOption("GUI Element Exists")]
+        [PropertyUISelectionOption("Error Occured")]
+        [PropertyUISelectionOption("Error Did Not Occur")]
+        [InputSpecification("Please select a condition type for the loop.")]
+        [SampleUsage("")]
+        [Remarks("")]
         public string v_LoopActionType { get; set; }
 
         [XmlElement]
-        [Attributes.PropertyAttributes.PropertyDescription("Additional Parameters")]
-        [Attributes.PropertyAttributes.PropertyUIHelper(Attributes.PropertyAttributes.PropertyUIHelper.UIAdditionalHelperType.ShowVariableHelper)]
-        [Attributes.PropertyAttributes.InputSpecification("Select the required comparison parameters.")]
-        [Attributes.PropertyAttributes.SampleUsage("n/a")]
-        [Attributes.PropertyAttributes.Remarks("")]
+        [PropertyDescription("Additional Parameters")]
+        [InputSpecification("Select the required comparison parameters.")]
```

Comment:
Doesn't match equivalent If Command

Reply:
I've notice this is a common thing among many your attribute changes. Please make them consistent

Reply:
done